spec/notification.js

```diff
@@ -92,6 +92,31 @@ describe('notification api', () => {
       });
     });
 
+    it('should send an email with an unsubscribe link', () => {
```

Comment:
@quis should we also change the wording in the test name to be consistent?

Reply:
```suggestion
    it('should send an email with a one click unsubscribe URL', () => {
```